### PR TITLE
analysis_options v1.9.0 compliance

### PR DESCRIPTION
**Pull request by domesticmouse**
PTAL @devoncarew

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # grinder.dart changes
 
+## 0.8.4
+- Update the code to be analysis_options v1.9.0 compliant
```
Comment (devoncarew) [inline CHANGELOG.md]: ==> `package:pedantic v1.9.0 compliant`

File: example/grind.dart [hunk 2]
```diff
@@ -3,21 +3,21 @@
 
 import 'package:grinder/grinder.dart';
 
-main(args) => grind(args);
+Future<dynamic> main(args) => grind(args);
 
 @Task('Initialize stuff.')
-init() {
-  log("Initializing stuff...");
+void init() {
```
Comment (devoncarew) [inline example/grind.dart]: I like the added use of types on API boundaries; definitely miss the use of types within a method.

File: CHANGELOG.md (hunk 1 above)
Comment (domesticmouse) [reply CHANGELOG.md]: Much better, thanks Devon. Done.

File: example/grind.dart (hunk 2 above)
Comment (domesticmouse) [reply example/grind.dart]: Yeah, agree

**Pull request merged**